src/ec2/resources/crontab

```diff
@@ -8,5 +8,6 @@
+56 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn51472302 --raw-folder-id  syn51449219 --glue-workflow bridge-downstream-mobile-toolbox-mtbwrj-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/mtbwrj/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/mtbwrj/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates --query "select * from {source_table} where createdOn > 1684825200000"
 35 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view syn51534381 --raw-folder-id syn51511828 --glue-workflow bridge-downstream-mobile-toolbox-pgdvpj-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/pgdvpj/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/pgdvpj/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates
+40 * * * * ec2-user docker run --rm 611413694531.dkr.ecr.us-east-1.amazonaws.com/bootstrap_trigger:latest python /root/BridgeDownstream/src/scripts/bootstrap_trigger/bootstrap_trigger.py --file-view  syn51534367 --raw-folder-id  syn51475701 --glue-workflow bridge-downstream-mobile-toolbox-vwrdjf-S3ToJsonWorkflow --ssm-parameter synapse-bridgedownstream-auth --diff-s3-uri 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/vwrdjf/parquet/dataset_sharedschema_v1/' --diff-parquet-field recordid --diff-s3-uri-2 's3://bridge-downstream-parquet/bridge-downstream/mobile-toolbox/vwrdjf/parquet/dataset_archivemetadata_v1/' --diff-parquet-field-2 recordid --diff-file-view-field recordId --drop-duplicates --query "select * from {source_table} where createdOn > 1684825200000"
```

Comment:
How did you pick the createdOn threshold?

Reply:
That's when we started getting records from the first build to not have any validation issues. Everything prior to that failed validation and wasn't getting processed (but was cluttering up our logs).

Reply:
Got it, so May 23rd 00:00 PT.